cbdr/cortex
# 10%: Environment Indicator

## Purpose:
* Environment indicator/banner (this will not appear on Production)
* Missing branding text fix (pages that defaulted to collapsed sidebar were missing Cortex branding text beside logo when sidebar was manually expanded)

## JIRA:
N/A

## Steps to Take On Prod
N/A

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
![screenshot from 2017-04-30 14-02-14](https://cloud.githubusercontent.com/assets/1045168/25567178/be4b8bf6-2dad-11e7-9996-c1f3ed085b44.png)
![screenshot from 2017-04-30 14-02-47](https://cloud.githubusercontent.com/assets/1045168/25567179/c0660808-2dad-11e7-80a7-c38f1a87aabb.png)

## QA Links:
http://web.cortex-1.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * Main page (right after login) with expanded sidebar
  * `ContentType` CRUD pages

* Steps to take
  * Login (`admin@cortexcms.org`/`welcome1`)
  * Ensure default, expanded sidebar displays:
    * Environment indicator with 'DEVELOPMENT' text
    * 'Cortex' text next to logo
  * Navigate to a `ContentType` CRUD page
  * Ensure default, collapsed sidebar displays:
    * Environment indicator with abbreviated 'DEV' text
    * 'Cortex' text next to logo once you manually expand sidebar
    * Environment indicator with full 'DEVELOPMENT' text after you manually expand sidebar

* Responsive considerations
  * Ensure mobile menu appears as expected, with correct branding text and environment indicator

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
